python/tvm/relay/frontend/onnx.py

```diff
@@ -5451,6 +5451,36 @@ def _impl_v16(cls, inputs, attr, params):
         )
 
 
+class Bernoulli(OnnxOpConverter):
+    """Operator converter for Bernoulli"""
+
+    @classmethod
+    def _impl_v15(cls, inputs, attr, params):
+        in_dtype = infer_type(inputs[0]).checked_type.dtype
+        assert in_dtype in [
+            "float32",
+            "float64",
+        ], "Only float input tensor is currently supported."
```

Comment:
ONNX has support for `float16`. However, this type is not supported here. Maybe it's worth pointing out the reason (`TODO`) why this data type is not currently supported at the ONNX front-end level?